app/controllers/charges_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ChargesController < ApplicationController
+	def new
```

Comment:
file deleted from the repository  https://github.com/fabriziobertoglio1987/railsmentor/commit/36c8a41b58caa421d02225ca4ef8d03344c32cdb